### PR TITLE
Add samze to reviews of cli

### PR DESCRIPTION
**Pull request by Samze**
👋 Back working on CloudFoundry and CLI at Broadcom, looking at eventually becoming an approver. I've worked on different bits of CF in the past (see https://github.com/cloudfoundry/community/pull/848) and have just started looking at CLI, e.g. :
- https://github.com/cloudfoundry/cli/pull/2921
- https://github.com/cloudfoundry/cli/pull/2881#discussion_r1600368379

### CODE REVIEW

**Comment by Gerg** [comment]
cc @cloudfoundry/wg-app-runtime-interfaces-cli-approvers for review

**Review by moleske — APPROVED** [review]

**Comment by beyhan** [comment]
@Gerg we have two thumps up/approval and this can be merged.

**Review by stephanme — APPROVED** [review]

**Comment by stephanme** [comment]
The pre-requisites for becoming a reviewer are met. Welcome as reviewer in the CLI area of ARI WG.

**Pull request merged**
